packages/fluentui/react-bindings/package.json

```diff
@@ -13,7 +13,7 @@
     "@fluentui/react-northstar-styles-renderer": "^0.62.0",
     "@fluentui/state": "^0.62.0",
     "@fluentui/styles": "^0.62.0",
-    "@uifabric/utilities": "^7.32.3",
+    "@fluentui/utilities": "^8.8.3",
```

Comment:
I'm quite sure this will mess up dependencies, and force v8 to build for v0 changes

Reply:
However it seems that there is already a dep on v8 in northstar

https://github.com/microsoft/fluentui/blob/2769e4b68c65eb8e06cf15384f51059e819b1fc5/packages/fluentui/react-northstar/package.json#L10

Reply:
hmm, I think getting rid of those in v0 would be not a small effort right ? maybe we can copy/paste ? wdyt

Reply:
I would like to avoid "build first" approach for N*. `@uifabric/utilities` was used directly from NPM before. Can we do the same?`

Reply:
Copy-paste is also an option.

Reply:
The hack is even a step uglier than for `@fluentui/dom-utilties` I needed to add a resolution because there is no major version under v8 that I can use for `@fluentui/utilities`

Reply:
ended up copy pasting because `@fluentui/utilities` also has a dependency on merge-styles